### DISCUSSION TITLE
In a multi-round conversation, when using a post request, how can I change the value of the input parameter after the second round?

### DISCUSSION THREAD

**Discussion by huu3301** [langgenius/dify#10036] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

In a multi-round conversation, the sys.query can change in each round, but the custom variable input cannot be changed, It can only be the value of the first input. How can I set an input variable whose value can change in different round of a multi-round conversation for post requests?

### 2. Additional context or comments

For example, in a multi-round conversation, I pass "aaa" as the router parameter for a post request in the first round and "bbb" as the router parameter in the second round. However, the reality is that the parameter in the second round is not updated and is still the value "aaa" of the first round.
![image](https://github.com/user-attachments/assets/592985f0-7814-4edb-bc6b-0d025c8df202)

**Comment by crazywoola** [comment]
You can use variable assigner to achieve this, which can make sure each time it will use the new value.

**Reply by ducanh997** [reply]
@crazywoola  I tried using the variable assigner, but it still didn't solve the issue. How did you implement using the variable assigner?

**Reply by hs354355279** [reply]
@huu3301 I've encountered this situation too. Is there any way to resolve it at the moment?

**Comment by trailblazer999** [comment]
when update to 1.4.0, it seems solved. i didn't find any changes mentioned it.
